core/src/test/java/org/infinispan/persistence/ClusterCacheLoaderTest.java

```diff
@@ -22,7 +22,7 @@
  * @author Mircea.Markus@jboss.com
  */
 @Test(groups = "functional", testName = "persistence.ClusterCacheLoaderTest")
-@InCacheMode({ CacheMode.INVALIDATION_SYNC, CacheMode.DIST_SYNC, CacheMode.REPL_SYNC, CacheMode.SCATTERED_SYNC })
+@InCacheMode({ CacheMode.INVALIDATION_SYNC/*, CacheMode.DIST_SYNC, CacheMode.REPL_SYNC, CacheMode.SCATTERED_SYNC*/ })
```

Comment:
CcaheModes need uncommenting :slightly_smiling_face:

Reply:
Yes, forgot to put them back in.